src/staging/src/components/park/parkAccessStatus.js

```diff
@@ -64,8 +65,8 @@ export default function ParkAccessStatus({ advisories }) {
       } else {
         // advisory is coming from explore page
         // get accessStatus based on precedence
-        let thisStatus = accessStatusList.filter(status => {
-          return status.node.precedence === advisory.accessStatus;
+        let thisStatus = accessStatusList.find(status => {
```

Comment:
Just need to update `thisStatus[0]` to `thisStatus` below since find returns just the item or null 🙂

Reply:
right again!